src/ai/backend/client/cli/admin/domain.py

```diff
@@ -62,7 +62,10 @@ def list(ctx: CLIContext) -> None:
 @click.option("-i", "--inactive", is_flag=True, help="New domain will be inactive.")
 @click.option("--total-resource-slots", type=str, default="{}", help="Set total resource slots.")
 @click.option(
-    "--allowed-vfolder-hosts", type=str, multiple=True, help="Allowed virtual folder hosts."
+    "--allowed-vfolder-hosts",
```

Comment:
We need to tell user how to fill out this parameter. After reading actual implementation it looks like the code expects string formatted as JSON syntax to be filled here, so let's tell user that way.